pheps/phep-9999.md

```diff
@@ -0,0 +1,171 @@
+
+# Rationale
+<a name="rationale"></a>
+Decisions for tiering levels, requirements for each tier, and benefits accrued at each tier are based on conversations with the community (bi-annual meetings, telecons, etc.), and are listed here as a starting point for more discussion, likely to be refined in the future. Initially, ideas were presented to the community in a pyramid format. To make the differences between tiers more visible and understandable, it has been transformed into a spreadsheet format. 
+
+# PyHC Package Tiering Specifications
+<a name="specification"></a>
+There are four tiers proposed in this PHEP: Copper, Bronze, Silver, and Gold. These would replace the current structure of "Core Packages", "Other Packages", and "Un-evaluated Packages". See the table below for requirements associated with each tier:
+
+|       | **Gold** | **Silver** | **Bronze** | **Copper** | 
+| :-: | :----------: | :-----------: | :------------: | :-------------: |
+| Self-Evaluation Status | Completed | Completed | Completed | Not Done |
+| PyHC Standard Grades | Mostly green, some yellow | Several yellow, no red | A couple red | N/A |
+| pyOpenSci Review Status | Completed | In progress | Not Started | Not Started |
+| PHEP 3 Compliant? | Yes | Yes | Yes | No |
+| HSSI Metadata Compliant? | Yes | Mostly | Partially | Bare Minimum Met |
+| Software License | Fully compliant and excludes NOSA license | Fully compliant, allows NOSA license | Has non-recommended license (e.g., GPL) | Has no license | 
+| DOI | Required | Required | Required | Not Required |
+| pip installable? | Yes | Yes | Yes | No |
+| conda installable? | Yes | No | No | No |
+
+Descriptions for each heading are as follows:
+   - Self Evaluation Status: indicates whether a package has completed a self evaluation against PyHC's standards
+   - PyHC Standard Grades: indicates status of each standards grouping within a package's self evaluation
+   - pyOpenSci Review Status: indicates status of a pyOpenSci review
+   - PHEP 3 Compliant?: indicates state of installation conflicts within the PyHC software environment (i.e. does the Package comply with PHEP 3)
+   - HSSI Metadata Compliant?: indicates state of compliance with the new Heliophysics Software Search Interface's (HSSI) metadata requirements; more specifically:
+      - Copper: Name of package, link to repository  
+      - Bronze: Copper + DOI, license, description, (publisher, publication year, authors -> needed to created DOI), mandatory fields for HSSI (TBD)
+      - Silver: Bronze + most recommended fields for HSSI (TBD)
+      - Gold: Silver + all recommended and some optional fields for HSSI (TBD)
+   - Software License: indicates level of software license compliance with the [PyHC Standards Document](https://doi.org/10.5281/zenodo.2529131) 
```

Comment:
Can this be treated like any other standard instead of making it a separate bullet here? I'm not sure about calling out NOSA. It is OSI-compliant but not FSF-compliant. There might be some issues if a package needs a NOSA dependency (or ships it alongside)--is that an immediate "can't be gold"? How is NASA doing at getting things relicensed? The [CDF license is somewhat ambiguous](https://lists.debian.org/debian-science/2010/12/msg00032.html) re DFSG and I don't know if it's been FSF or OSI evaluated (I don't think it has).

Reply:
The process is ...interesting at NASA to get things relicensed. At Goddard, the leadership knows the problem in detail and is having significant problems moving forward. Hopefully a leap-frog moment is on the horizon.... With that said, NOSA is worth calling out because it prohibits contribution except through a long registration process.

Reply:
I was suggested by @rebeccaringuette to remove NOSA as a possibility due to the reasoning she gave here. It seems reasonable to me. Might even help NASA to improve that license (if that's a possibility) if a software community directly calls out _not_ using it. 

Perhaps this bullet could go away if we get standards PHEPs in to point to (where this NOSA issue could be put)? I do worry though about this taking a long time to enact though if it's dependent on other PHEPs all getting written, approved, etc.

Reply:
I wouldn't think we need to hold this up until all standards PHEPs are done. If we have something like "when new standards-track PHEPs are approved packages have 6 (12?) months to self-evaluate and update their tiers", then we could in theory approve this with no new standards lined up. There's going to be a transition period regardless.